erigon-lib/state/aggregator.go

```diff
@@ -180,20 +177,16 @@ func NewAggregator(ctx context.Context, dirs datadir.Dirs, aggregationStep uint6
 	//if a.d[kv.GasUsedDomain], err = NewDomain(cfg, aggregationStep, "gasused", kv.TblGasUsedKeys, kv.TblGasUsedVals, kv.TblGasUsedHistoryKeys, kv.TblGasUsedHistoryVals, kv.TblGasUsedIdx, logger); err != nil {
 	//	return nil, err
 	//}
-	idxCfg := iiCfg{salt: salt, dirs: dirs, db: db}
-	if a.logAddrs, err = NewInvertedIndex(idxCfg, aggregationStep, "logaddrs", kv.TblLogAddressKeys, kv.TblLogAddressIdx, nil, logger); err != nil {
+	if err := a.registerII(kv.LogAddrIdxPos, salt, dirs, db, aggregationStep, "logaddrs", kv.TblLogAddressKeys, kv.TblLogAddressIdx, logger); err != nil {
```

Comment:
I think can use `kv.LogAddrIdxPos.String()` instead of `"logaddrs"`

Reply:
unfortunately not, it’s a number constant

Reply:
@wmitsuda see Domain type. It has String() method

Reply:
It’s enum

Reply:
done by: https://github.com/ledgerwatch/erigon/pull/10423

actually in this specific line the string refers to the file prefix, but I decided to extract those into a `files.go` constant file like `tables.go` for snapshot-ish constants.

but there is another places in the code in which log constants are used and there I turned into an enum method.